crypto/evp/p_lib.c

```diff
@@ -2495,3 +2498,172 @@ int EVP_PKEY_get_field_type(const EVP_PKEY *pkey)
     return 0;
 }
 #endif
+
+/*- All methods below can also be used in FIPS_MODULE */
+
+int EVP_SKEY_import(EVP_SKEY *skey, const OSSL_PARAM *params)
```

Comment:
`crypto/evp/p_lib.c` is (or should be) about `EVP_PKEY`.  To stay on topic, would you mind creating a separate file for purely `EVP_SKEY` stuff?  `s_lib.c` or `skey_lib.c`, for example?

Reply:
Will do on next rebase

Reply:
Done